caffe2/python/memonger.py

```diff
@@ -798,15 +798,29 @@ def canonical_name(blob):
             op.output[i] = canonical_name(output)
 
 
-
 def apply_recurrent_blob_assignments(op, blob_assignments, canonical_name):
```

Comment:
I might be wrong, but I think you might need to cover few more parameter for potential remapping. Based on the source from caffe2/python/recurrent.py the following args can potentially contain blob references:

Forward: alias_src, alias_dst, recurrent_states, link_internal, link_external, timestep
Backward: param, backward_link_internal, backward_link_external, recompute_blobs_on_backward, param_grads.

I think it's good to cover all of those or explicitly forbid them to be present (if we don't want to support backward pass).

Also cc @aazzolini, who is original author of RecurrentNetwork operator (in case if I'm missing something).

Reply:
This is to fix an error we found in our model, so it's tested there.
Regarding other attributes, I remember there're definite some of them.
But it's too long ago and I don't remember those details.
All I remember is "they didn't cause trouble in our model, so I'd rather not touch them to make sure the correctness of my PR".

Reply:
For backward, it's the same reason. We were doing inferencing at that time, so not sure about those things outside of our code coverage.